pom.xml

```diff
@@ -10,7 +10,7 @@
 	</parent>
 	<groupId>ca.on.oicr.icgc.argo</groupId>
 	<artifactId>program-service</artifactId>
-	<version>1.0.7-SNAPSHOT</version>
+	<version>1.1.1</version>
```

Comment:
I left this as it was. But I can change it to `1.0.0` since we have no tags with versions to begin with